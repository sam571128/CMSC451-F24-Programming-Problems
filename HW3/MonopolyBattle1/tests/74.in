10 54 704888572
4 3
1 10
7 5
9 6
6 5
4 8
2 2
6 2
9 8
5 5
7 9
4 5
3 6
7 7
7 10
1 2
6 10
9 1
9 2
5 8
6 4
3 10
10 3
7 6
8 1
4 4
2 8
6 8
8 10
6 7
7 3
7 4
7 1
3 7
6 6
7 2
5 6
8 2
9 3
3 9
6 3
2 7
10 5
2 6
4 10
4 2
8 6
3 4
3 5
5 1
10 8
8 9
3 8
4 9
