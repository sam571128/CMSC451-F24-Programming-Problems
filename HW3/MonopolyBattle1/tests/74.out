25 16 23 23 23 23 30 23 30 23 